### DISCUSSION TITLE
onthisday/12/12-18/

### DISCUSSION THREAD

**Discussion by giscus** [FanzOfLenaZavaroni/fanzoflenazavaroni.github.io#33] [General]
# onthisday/12/12-18/

Details for On This Day are genarated by the posts added to the website so the content is subject to changes/updates over time.

https://fanzoflenazavaroni.github.io/onthisday/12/12-18/

**Comment by StephenMolloy1** [comment]
The beginning and the ending have appeared on TV Ark, no footage of Lena, but she is listed on the closing credits.
https://tvark.org/crackerjack-4

**Comment by FanzOfLenaZavaroni** [comment]
Thanks for the link at least it confirms we have the right date for the episode in which Lena was on.

I just checked out the [BFI Collections Website](https://collections-search.bfi.org.uk/web/Details/ChoiceFilmWorks/150575327) seem that they only have a copy on VHS and they have yet to add who the guests on the show were only listing the regular cast members.

Searching of the BFI Collections is very unreliable in the end to get anything to come up I had to search by just "Crackerjack" and then go through the results myself to find the episode and sadly the way the data is presented means its currently not possible to link to a specific section of the content.

Using Google or Microsoft Bing to do a site search is also disappointing as very few results show up I would have thought that as the BFI Collections Website has been running for over a year now more of its content would have been crawled by the search engines but the limit may also be due to the poor coding structure of the BFI site as the search results all look very much the same which may be leading the search engines to ignore most of the content treating it all as duplicates, rant over!

**Reply by StephenMolloy1** [reply]
The BFI seem to have nearly everything she appeared in, The problem seems to be that a lot of it is in obsolete formats and by the look of things it is going to take them years to convert them.

**Reply by FanzOfLenaZavaroni** [reply]
I'm hoping the fact that the BFI states that this Videotape can be requested for access means that a currently unlisted format is in their archive (They have a disclaimer that not all details have been added to the Collections website and that details will be added as and when possible and that details may not be free of errors).

Yes! it's great to know most of Lena's TV work is archived but sadly for how much Longer The VHS players are not going to function indefinitely and an archive of unplayable content is pretty much pointless. This is why fan owned content is so important to Lena's legacy.

### WAFFLE:
As the BFI was created around the same time as the BBC it must mean they have a mass of content that is in need of preservation and sadly some of it will become lost over time due to the large amount of it and the cost of preservation. While we fans want to see Lena's shows not only preserved but ideally made available to watch in our homes. The reality is other content is going to be considered of more value before many shows including Lena's.

Sadly the BBC has no legal obligation to maintain an archive, the only obligation the BBC have is that if they do not want to keep a copy of a programme themselves they must offer a copy to the BFI and the BFI will decide if they want to take up the offer or not.

It has always made me question why the BFI was not named "The British Film & Television Institute" but I guess back when the BBC and BFI were set-up all TV was on film plus it meant if TV was a flop the BFI could go on archiving films without the need to rename.

**Reply by StephenMolloy1** [reply]
Even fewer radio shows seem to have been recorded, which is a shame as she did a lot on Radio 2, and some local radio interviews in places like Blackpool.

**Reply by FanzOfLenaZavaroni** [reply]
Yes! Radio seems to get over looked the BBC does have an archive of Radio content but a list of what it has is not ready available. Some content is listed on the [Radio - BBC Archive](https://www.bbc.co.uk/archive/radio/z6cqgwx#:~:text=Since%202008%20the%20BBC's%20Radio,million%20recordings%20and%20growing%20daily.) The British Library Sound Archive also has Radio Broadcasts Archived but what it has I don't know and at this time the website is of line, but heres the link anyway: [BL radio-broadcast-recordings](https://www.bl.uk/collection-guides/radio-broadcast-recordings).

Sites like mixcloud would be our best option to get to hear Lena related content but to date I have yet to find anything on there apart from what you and I have both found which is the "Top 20 with Tom Browne" and some of them have been taken down.
